docs/api.rst

```diff
@@ -187,7 +187,29 @@ runEffects
 
   runEffects :: Stream a -> Scheduler -> Promise void
 
-Activate an event stream, and consume all its events.
+Activate a :ref:`Stream` and consume all its events.
+
+.. _runStream:
+
+runStream
+`````````
+
+.. code-block:: haskell
+
+  runStream :: Sink a -> Scheduler -> Stream a -> void
+
+Run a :ref:`Stream`, sending all events to the provided :ref:`Sink`.  The Stream's :ref:`Time` values come from the provided :ref:`Scheduler`.
+
+.. _runStreamWithLocalTime:
+
+runStreamWithLocalTime
+``````````````````````
+
+.. code-block:: haskell
+
+  runStreamWithLocalTime :: Sink a -> Scheduler -> Time -> Stream a -> void
+
+Run a :ref:`Stream`, sending all events to the provided :ref:`Sink`.  The Stream will have localized :ref:`Time` values, whose origin (i.e. time 0) is at the specified Time on the provided :ref:`Scheduler`.
```

Comment:
Comma after "i.e.". So:

... whose origin (i.e., time 0)

The same goes for e.g.

Note that the comma after the abbreviations is American style. The British style omits the comma.

Reply:
Fixed in 034e260